### PR TITLE
feat(Calendar): add a calendar in the drawer CalendarPage

### PR DESCRIPTION
**Pull request by MathisCaugant**
setup a calendar

### CODE REVIEW

**Review by TurpaultM — DISMISSED** [review]
i approve it

File: SurvivorNext/Pages/CalendarPage.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        'Septembre',
+        'Octobre',
+        'Novembre',
+        'Décembre'
+    ],
+    monthNamesShort: ['Janv.', 'Févr.', 'Mars', 'Avril', 'Mai', 'Juin', 'Juil.', 'Août', 'Sept.', 'Oct.', 'Nov.', 'Déc.'],
+    dayNames: ['Dimanche', 'Lundi', 'Mardi', 'Mercredi', 'Jeudi', 'Vendredi', 'Samedi'],
+    dayNamesShort: ['Dim.', 'Lun.', 'Mar.', 'Mer.', 'Jeu.', 'Ven.', 'Sam.'],
+    today: "Aujourd'hui"
+};
```
Comment (MATIFIREofficiel) [inline SurvivorNext/Pages/CalendarPage.js:23]:
```suggestion
LocaleConfig.locales['en'] = {
    monthNames: [
        'January',
        'February',
        'March',
        'April',
        'May',
        'June',
        'July',
        'August',
        'September',
        'October',
        'November',
        'December'
    ],
    monthNamesShort: ['Jan.', 'Feb.', 'March', 'April', 'May', 'June', 'July', 'Aug.', 'Sept.', 'Oct.', 'Nov.', 'Dec.'],
    dayNames: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
    dayNamesShort: ['Dim.', 'Lun.', 'Mar.', 'Mer.', 'Jeu.', 'Ven.', 'Sam.'],
    today: "Today"
};
```

**Review by MATIFIREofficiel — APPROVED** [review]
Accept the suggestion and it's okay

**Review by MATIFIREofficiel — APPROVED** [review]
Okay

File: SurvivorNext/Pages/CalendarPage.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December'
+    ],
+    monthNamesShort: ['Jan.', 'Feb.', 'March', 'April', 'May', 'June', 'July', 'Aug.', 'Sept.', 'Oct.', 'Nov.', 'Dec.'],
+    dayNames: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
+    dayNamesShort: ['Dim.', 'Lun.', 'Mar.', 'Mer.', 'Jeu.', 'Ven.', 'Sam.'],
```
Comment (MATIFIREofficiel) [inline SurvivorNext/Pages/CalendarPage.js:21]:
```suggestion
    dayNamesShort: ['Sun.', 'Mon.', 'Tue.', 'Wed.', 'Thu.', 'Fru.', 'Sat.'],
```

**Review by MATIFIREofficiel — CHANGES_REQUESTED** [review]
Accept suggestion

File: SurvivorNext/Pages/CalendarPage.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December'
+    ],
+    monthNamesShort: ['Jan.', 'Feb.', 'March', 'April', 'May', 'June', 'July', 'Aug.', 'Sept.', 'Oct.', 'Nov.', 'Dec.'],
+    dayNames: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
+    dayNamesShort: ['Sun.', 'Mon.', 'Tue.', 'Wed.', 'Thu.', 'Fru.', 'Sat.'],
```
Comment (MATIFIREofficiel) [inline SurvivorNext/Pages/CalendarPage.js:21]:
```suggestion
    dayNamesShort: ['Sun.', 'Mon.', 'Tue.', 'Wed.', 'Thu.', 'Fri.', 'Sat.'],
```

**Review by MATIFIREofficiel — CHANGES_REQUESTED** [review]

**Review by MATIFIREofficiel — APPROVED** [review]
Okay

**Review by TurpaultM — APPROVED** [review]
yes

**Pull request merged**